for_production.sh

```diff
@@ -26,21 +26,17 @@ source ocr.sh
 
 init "$@"
 
-# run the workflow script on the controller non-interactively and log its output locally
+# run the workflow script on the Controller non-interactively and log its output locally
 # subsequently validate and postprocess the results
 # do all this in a subshell in the background, so we can return immediately
 (
     pre_process_to_workdir
-	
-	# TODO: copy the data explicitly from manager to controller here
-    # e.g. `rsync -avr "$WORKDIR" --port $CONTROLLERPORT ocrd@$CONTROLLERHOST:/data`
+
+    pre_sync_workdir
 
     ocrd_exec ocrd_import_workdir ocrd_validate_workflow ocrd_process_workflow
 
-    # TODO: copy the results back here
-    # e.g. `rsync -avr --port $CONTROLLERPORT ocrd@$CONTROLLERHOST:/data/"$WORKDIR" "$WORKDIR"`
-    # maybe also schedule cleanup (or have a cron job delete dirs in /data which are older than N days)
-    # e.g. `ssh --port $CONTROLLERPORT ocrd@$CONTROLLERHOST rm -fr /data/"$WORKDIR"`
+    post_sync_workdir
 
     post_process_validate_workdir
```

Comment:
Should we rename  `post_process_validate_workdir` to `post_validate_workdir` for more continuity? Maybe   `pre_process_to_workdir ` but there i think it's necessary to know what happends.

Reply:
Agreed!

Reply:
Done